pcdsdevices/daq.py

```diff
@@ -168,7 +171,7 @@ def begin(self, events=None, duration=None, use_l3t=None, controls=None,
                      events, duration, wait)
         begin_status = self.kickoff(events=events, duration=duration,
                                     use_l3t=use_l3t, controls=controls)
-        begin_status.wait()
+        begin_status.wait(timeout=BEGIN_TIMEOUT)
```

Comment:
Think there is a use case for having a `kwarg` for this wait period? Sometimes you might want to check quickly, do something else, check again e.t.c

Reply:
This is what the kickoff function is for, you can call it directly and check on the status object. Begin strives to fill the niche of "once I begin the daq I expect it to have been started." In practice, the status is marked done in about 0s in most cases, in about 0.5 seconds if you've just ended a really short run, and after the default timeout length if the daq is derping out. That is, if it will succeed it will succeed quickly.

Reply:
Sounds good. `kickoff` it is!